Maskbook/Scene/Wallet/TokenDetail/View/TransactionCell.swift

```diff
@@ -314,8 +314,22 @@ extension TransactionCell {
             guard let gasPrice = txToSpeedUp.transactionInfo?.gasPrice else { return }
             guard let gasLimit = txToSpeedUp.transactionInfo?.gaslimit else { return }
             
-            let confirmModel = SendConfirmViewModel(selectedToken: txToSpeedUp.transactionInfo?.token, gasPrice: gasPrice * BigUInt(1.2), gasLimit: gasLimit, gasFeeNetModel: txToSpeedUp.transactionInfo?.gasNetModel)
+            var gasNetModel = txToSpeedUp.transactionInfo?.gasNetModel
+
+            if let maxFeePerGas = txToSpeedUp.transactionInfo?.gasNetModel?.suggestedMaxFeePerGas{
+                let gwei = NSDecimalNumber(string: maxFeePerGas).multiplying(by: NSDecimalNumber(string: "1.2"))
+                let roundBehavior = NSDecimalNumberHandler(roundingMode: .plain, scale: 0, raiseOnExactness: false, raiseOnOverflow: false, raiseOnUnderflow: false, raiseOnDivideByZero: false)
+                gasNetModel?.suggestedMaxFeePerGas = gwei.rounding(accordingToBehavior: roundBehavior).stringValue
+            }
             
+            if let maxPriorityFee = txToSpeedUp.transactionInfo?.gasNetModel?.suggestedMaxPriorityFeePerGas{
+                let gwei = NSDecimalNumber(string: maxPriorityFee).multiplying(by: NSDecimalNumber(string: "1.2"))
```

Comment:
Init a NSDecimalNumber with a `number` would be better than using a `string` IMO